gstreamer-sdp/src/auto/m_i_k_e_y_decrypt_info.rs

```diff
@@ -0,0 +1,11 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir @ 7f5a2b5)
+// from gir-files (https://github.com/gtk-rs/gir-files @ ???)
+// DO NOT EDIT
+
+use ffi;
+use glib::translate::*;
+use glib_ffi;
+use std::mem;
+use std::ptr;
+
+pub struct MIKEYDecryptInfo(ffi::GstMIKEYDecryptInfo);
```

Comment:
This probably should get some field accessors at least, maybe a constructor. Not sure how it is actually used :)